processor/src/test/java/org/mapstruct/ap/test/bugs/_880/Poodle.java

```diff
@@ -16,30 +16,20 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.option;
-
+package org.mapstruct.ap.test.bugs._880;
 
 /**
- * Helper class for dealing with {@link Options}.
- *
  * @author Andreas Gudian
+ *
  */
-public class OptionsHelper {
+public class Poodle {
```

Comment:
So... It took me some time to put `Poodle` and `Core` together.... Its a literal translation of what means "The heart of the matter` or, on Dutch `De kern van de zaak`..  :)..

Looks a case of: "a strange duck in the pond"    :)